config/application.rb

```diff
@@ -13,12 +13,12 @@ class Application < Rails::Application
     config.time_zone = 'UTC'
     config.middleware.use Rack::Attack
     config.browserify_rails.force = true
-    config.browserify_rails.commandline_options = '-t [ babelify --presets [ es2015 ] ]'
+    config.browserify_rails.commandline_options = '-t [ babelify --presets [ es2015 ] --plugins [ transform-es3-member-expression-literals transform-es3-property-literals ]'
```

Comment:
looks like code climate is complaining about the line being too long. Also is it missing a `]`?

Maybe we could do something like:

```ruby
config.browserify_rails.commandline_options = %w(
  -t [
    babelify --presets [ es2015 ] 
    --plugins [ transform-es3-member-expression-literals transform-es3-property-literals ]
  ]
).join(' ')
```